lib/ama_validators/name_format_validator.rb

```diff
@@ -0,0 +1,7 @@
+class NameFormatValidator < ActiveModel::EachValidator
+  def validate_each(object, attribute, value)
+    unless value =~ /\A[a-zA-Z]+\z/
```

Comment:
Gwyneth had mentioned something about D'arcy or something where it caused things to break or crash.